packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4740,8 +4740,12 @@ class Flex extends MultiChildRenderObjectWidget {
     this.textBaseline, // NO DEFAULT: we don't know what the text's baseline should be
     this.clipBehavior = Clip.none,
     this.spacing = 0.0,
+    this.inversedChildren = false,
     super.children,
-  }) : assert(!identical(crossAxisAlignment, CrossAxisAlignment.baseline) || textBaseline != null, 'textBaseline is required if you specify the crossAxisAlignment with CrossAxisAlignment.baseline');
+  }) : assert(!identical(crossAxisAlignment, CrossAxisAlignment.baseline) || textBaseline != null, 'textBaseline is required if you specify the crossAxisAlignment with CrossAxisAlignment.baseline'),
+        super(
+        children: inversedChildren ? children?.reversed.toList() : children,
+      );
```

Comment:
This results in a compile-time error, since `children?.reversed.toList()` is not a valid `const` constructor argument.